src/components/people/SelectedPersonResultCard.js

```diff
@@ -94,14 +65,14 @@ const labelMap = Map({
 const SelectedPersonResultCard = ({ index, person } :Props) => (
   <Card>
     <NoPaddingCardSegment>
-      <PictureWrapper>
-        {renderPersonPicture(person)}
-        {
-          index !== undefined && (
+      <PersonPicture person={person} size="xl" />
+      {
+        index !== undefined && (
+          <UtilizerTagWrapper>
             <UtilizerTag>{`#${index} Utilizer`}</UtilizerTag>
```

Comment:
Does the default -1 index make sense here?

Reply:
To be honest, I didn't even think about this. I just wanted to upgrade the codebase to use the new `PersonPicture` component.